src/tools/remote_worker/src/main/java/com/google/devtools/build/remote/ExecutionServer.java

```diff
@@ -188,7 +188,14 @@ private ActionResult execute(Action action, Path execRoot)
       FileSystemUtils.createDirectoryAndParents(file.getParentDirectory());
       outputs.add(file);
     }
-    // TODO(olaola): support output directories.
+    for (String output : action.getOutputDirectoriesList()) {
```

Comment:
An integration testing both the client and remote worker implementation would be great!